pages/index.js

```diff
@@ -72,7 +72,7 @@ const Home = () => {
 
   const [universalLink, embedCode] = useMemo(() => {
     console.log('generating links and embed code')
-    const universalLink = getUniversalLink([...urls])
+    const universalLink = getUniversalLink(urls.map((e) => e.url))
```

Comment:
The only relevant change is this line here.

Reply:
👍